package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "dev": "razzle start",
     "build": "razzle build",
+    "eslint": "eslint --ext .js,jsx ./src --fix",
```

Comment:
Comments:
- Can this script be called something generic? i.e. `lint`.
- We shouldn't automatically fix errors with `--fix`.
- Can linting be ran against more files than those inside `./src`?
- Can linting be ran against more than just `js` and jsx` files?

Reply:
Okay, should be fine for now. We can change down the road if needed.

Reply:
nice one